src/test/attendees.test.ts

```diff
@@ -174,24 +181,29 @@ describe('Attendees resource', () => {
 
       statusCode = res.status
       contentType = res.header['content-type']
+      authenticateHeader = res.header['www-authenticate']
       response = res.body
 
       createdAttendee = await MongoDB.Attendees.findbyAttendeeId(attendee.attendeeid)
     })
 
-    it('should respond with status code 403 Forbidden', () => {
-      assert.strictEqual(statusCode, 403)
+    it('should respond with status code 401 Unauthorised', () => {
+      assert.strictEqual(statusCode, 401)
+    })
+
+    it('should respond with WWW-Authenticate header for basic realm "Admin access"', () => {
+      assert.strictEqual(authenticateHeader, 'Basic realm="Admin access"')

```

Comment:
Yeah it's just a function. As I mention in the PR, I wanted to extract this to some sort of archetype test that performs this check for free.